docs/source/UserGuide/commands.rst

```diff
@@ -1,12 +1,19 @@
+.. _commands_reference:
+
 ==================
 Commands Reference
 ==================
 
-This page is the work in progress. See more material here soon !
+This page is the work in progress.
+
+The shorthand section contains short commands specifically developed for the interactive use of a DIRAC UI.
+To be able to use these commands, please check the :ref:`dirac_tutorials` section for an
```

Comment:
```suggestion
To be able to use these commands, please check the :ref:`shorthand_commands` section for an
```

Reply:
This update fails the checks - I don't think any of the individual subsections in the commands reference section have labels.